src/client/datascience/notebook/notebookControllerManager.ts

```diff
@@ -575,7 +598,6 @@ export class NotebookControllerManager implements INotebookControllerManager, IE
                         this.disposables,
                         this.languageService,
                         this.workspace,
-                        this.isLocalLaunch ? 'local' : 'remote',
```

Comment:
Yay, no need, whether it remote or local, is derived from the kernel connection metadata